app/services/course_creation_service.rb

```diff
@@ -0,0 +1,96 @@
+class CourseCreationService
```

Comment:
Any service prefixed with `Course` should probably utilise the namespace/directory we have here?

Reply:
Done